templates/sqoop-parquet-hdfs-hive-merge/Makefile.meta

```diff
@@ -0,0 +1,42 @@
+{#  Copyright 2017 Cargill Incorporated
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. #}
+
+first-run-all: {%- for table in tables %} first-run-{{ table.id }} {%- endfor %}
+
+{%- for table in tables %}
+first-run-{{ table.id }}:
+	$(MAKE) first-run -C {{ table.id }}
+{%- endfor %}
+
+update-all: {%- for table in tables %} update-{{ table.id }} {%- endfor %}
+
+{%- for table in tables %}
+update-{{ table.id }}:
+	$(MAKE) update -C {{ table.id }}
+{%- endfor %}
+
+clean-all:
+{%- for table in tables %}
+	$(MAKE) clean -C {{ table.id }}
+{%- endfor %}
+
+{%- for table in tables %}
+integration-test-{{ table.id }}:
+	$(MAKE) integration-test -C {{ table.id }}
+{%- endfor %}
+
+integration-test-all:
+{%- for table in tables %}
+	$(MAKE) integration-test -C {{ table.id }}
```

Comment:
switch to TAB

Reply:
seems to be tab already:
integration-test-all:^M$
{%- for table in tables %}^M$
^I$(MAKE) integration-test -C {{ table.id }}^M$
{%- endfor %}^M$